cmake/CMakeLists.txt

```diff
@@ -684,6 +686,11 @@ if (onnxruntime_USE_DML)
   include(dml)
 endif()
 
+# allow the caller to tell us where the nuget packages are going
+if ("${REPO_PACKAGE_LOCATION}" STREQUAL "")
+    set(REPO_PACKAGE_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/packages)
+endif()
+
```

Comment:
what was this for?

Reply:
this was for the dml nuget package to work both from the github side and from the WindowsAI side.   it was since removed since the windowsAI side is using DML built from there and will not use the nuget.

---
In reply to: [354032371](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354032371) [](ancestors = 354032371)